### PR TITLE
Add some missing info on recent StackExchange.Redis versions

### PR DESCRIPTION
**Pull request by clearlydefinedbot**
**Type:** Missing

**Summary:**
Add some missing info on recent StackExchange.Redis versions

**Details:**
+License and commit details

**Resolution:**
Based on a review of the github repository history for StackExchange.Redis.

**Affected definitions**:
- [StackExchange.Redis 2.1.28](https://clearlydefined.io/definitions/nuget/nuget/-/StackExchange.Redis/2.1.28)

### CODE REVIEW

**Comment by LandOfBliss** [comment]
Looked in the "Files" section on the ClearlyDefined definition.  License file indicates MIT
Nuget license field indicates MIT
Source code project has MIT license
Conclusion: Declared license is MIT

File: curations/nuget/nuget/-/StackExchange.Redis.yaml [hunk 1]
```diff
@@ -42,24 +42,77 @@ revisions:
...
+      sourceLocation:
+        name: stackexchange.redis
+        namespace: stackexchange
+        provider: github
+        revision: 4a0d3538da09fc1ddba120efaaefd5787533a1e0
+        type: git
+        url: 'https://github.com/stackexchange/stackexchange.redis/commit/4a0d3538da09fc1ddba120efaaefd5787533a1e0'
+    licensed:
+      declared: MIT
+  2.1.28:
```
Comment (capfei) [inline curations/nuget/nuget/-/StackExchange.Redis.yaml:98]: This commit goes to v2.1.30.

File: curations/nuget/nuget/-/StackExchange.Redis.yaml [hunk 2]
```diff
@@ -42,24 +42,77 @@ revisions:
...
+  2.0.558:
+    described:
+      releaseDate: '2019-04-03'
+      sourceLocation:
+        name: stackexchange.redis
+        namespace: stackexchange
+        provider: github
+        revision: v2.0.558
+        type: git
+        url: 'https://github.com/stackexchange/stackexchange.redis/commit/v2.0.558'
```
Comment (capfei) [inline curations/nuget/nuget/-/StackExchange.Redis.yaml:72]: This is not a valid link. The repo does not have a tagged version for 2.0.558.

File: curations/nuget/nuget/-/StackExchange.Redis.yaml [hunk 3]
```diff
@@ -42,24 +42,77 @@ revisions:
...
+      declared: MIT
+  2.1.28:
+    described:
+      sourceLocation:
+        name: stackexchange.redis
+        namespace: stackexchange
+        provider: github
+        revision: 5c3036ab7aebd647f3dd3d89f07e02d4c2a48d5e
+        type: git
+        url: 'https://github.com/stackexchange/stackexchange.redis/commit/5c3036ab7aebd647f3dd3d89f07e02d4c2a48d5e'
```
Comment (capfei) [inline curations/nuget/nuget/-/StackExchange.Redis.yaml:106]: This commit goes to v2.1.30.

File: curations/nuget/nuget/-/StackExchange.Redis.yaml [hunk 4]
```diff
@@ -42,24 +42,77 @@ revisions:
...
       declared: MIT
   2.0.519:
     licensed:
       declared: MIT
+  2.0.544:
+    licensed:
+      declared: MIT
+  2.0.558:
+    described:
+      releaseDate: '2019-04-03'
```
Comment (capfei) [inline curations/nuget/nuget/-/StackExchange.Redis.yaml:65]: This is not the publish date for this version. 2.0.558 is dated as 3/7/2019.

File: curations/nuget/nuget/-/StackExchange.Redis.yaml (hunk 2 above)
Comment (ariel11) [reply curations/nuget/nuget/-/StackExchange.Redis.yaml:72]: @capfei - I agree with deleting the url from the PR.

File: curations/nuget/nuget/-/StackExchange.Redis.yaml (hunk 4 above)
Comment (ariel11) [reply curations/nuget/nuget/-/StackExchange.Redis.yaml:65]: @capfei - am I missing something, as I don't see some of these package versions actually published on Nuget.org.  I do not see a published package for 2.0.544 or 2.0558 or 2.0.510?

File: curations/nuget/nuget/-/StackExchange.Redis.yaml (hunk 3 above)
Comment (ariel11) [reply curations/nuget/nuget/-/StackExchange.Redis.yaml:106]: Agree - I see the same thing on row 3 here https://github.com/stackexchange/stackexchange.redis/commit/5c3036ab7aebd647f3dd3d89f07e02d4c2a48d5e

File: curations/nuget/nuget/-/StackExchange.Redis.yaml (hunk 4 above)
Comment (capfei) [reply curations/nuget/nuget/-/StackExchange.Redis.yaml:65]: They are unlisted:

- https://www.nuget.org/packages/StackExchange.Redis/2.0.510
- https://www.nuget.org/packages/StackExchange.Redis/2.0.544
- https://www.nuget.org/packages/StackExchange.Redis/2.0.558

Comment (capfei) [reply curations/nuget/nuget/-/StackExchange.Redis.yaml:65]: Unlisted versions do not show in the version list on NuGet.org so I have to add the version ID to the end of the address, like I did above.

**Pull request merged**